test/unit/lodash.spec.js

```diff
@@ -0,0 +1,568 @@
+                expect(_.pad.apply(_, params)).to.eql('._._.some long sweet text._._.');
+            }
+        );
+    });
+
+    describe('padEnd', function () {
+        before(function () {
+            params = datasets.padEnd();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+            expect(params[1]).to.be.above(params[0].length);
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(params[2]).to.be.a('string');
+            }
+        });
+        it('should return converted string with padding characters on right', function () {
+                expect(_.padEnd.apply(_, params)).to.eql('some another long sweet text.*.*.*.*');
+            }
+        );
+    });
+
+    describe('padStart', function () {
+        before(function () {
+            params = datasets.padStart();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+            expect(params[1]).to.be.above(params[0].length);
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(params[2]).to.be.a('string');
+                expect(params[2]).to.have.length.above(0);
+            }
+        });
+        it('should return converted string with padding characters on left', function () {
+                expect(_.padStart.apply(_, params)).to.eql('|_|_|_|_|_|_try again some long sweet text');
+            }
+        );
+    });
+
+    describe('parseInt', function () {
+        before(function () {
+            params = datasets.parseInt();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length.within(1, 2);
+            if (params[1] !== undefined && params.length === 2) {
+                expect(Number.isInteger(params[1])).to.eql(true);
+            }
+        });
+        it('should return converted string to integer', function () {
+                expect(_.parseInt.apply(_, params)).to.eql(8);
+            }
+        );
+    });
+
+    describe('repeat', function () {
+        before(function () {
+            params = datasets.repeat();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+        });
+        it('should return new repeated string', function () {
+                expect(_.repeat.apply(_, params)).to.eql('repeat me!repeat me!repeat me!repeat me!');
```

Comment:
In my opinion, without looking to this file, that solution is impossible to reveal